.github/dependabot.yml

```diff
@@ -2,6 +2,14 @@ version: 2
 updates:
   # Maintain dependencies for GitHub Actions
   - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "docker"
     directory: "/"
     schedule:
       interval: "daily"
```

Comment:
Can we fix newline?

Reply:
Indeed, I also added some comments for the new addiitions.